app/views/admin/merchants/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="page-title">Admin Merchants Edit</div> 
 <br><br>
-<%= form_with(model: @merchant, url: admin_merchant_path(@merchant), local: true, data: { turbo: false }) do |f| %>
+<%= form_with(model: @merchant, url: admin_merchant_path(@merchant), local: true, data: { turbo: false }, params: { merchant: :name}) do |f| %>
```

Comment:
I think that since `:name `is already being called inside the form and the model is preset, the code would likely still work for this without the  `params: { merchant: :name}` on line 3. But it also makes it clear that you need this info to be passed back from the form.